src/core/core/events.js

```diff
@@ -0,0 +1,34 @@
+import EventEmitter from 'eventemitter2'
```

Comment:
outside of the scope, the goal was to merge and try to preserve history, not to remove dead code. However, I have added an issue for it.